src/local_neighborhood.py

```diff
@@ -33,16 +33,16 @@ def generate_inputs(data, filename_map):
             node_df.loc[node_df['targets']==True, 'prize'] = 1.0
         else:
             raise ValueError("LocalNeighborhood requires nodes and networks")
-
+        
         # input node information
-        node_df.to_csv(filename_map["nodes"],sep="\t",index=False,columns=["#Node","Node type"])
+        node_df.to_csv(filename_map["nodes"],index=False,columns=["node"])
 
-        #For now we assume all input networks are undirected until we expand how edge tables work
+        # For now we assume all input networks are undirected until we expand how edge tables work
         edges_df = data.get_interactome()
         edges_df.to_csv(filename_map['network'],sep='|',index=False,columns=['vertex1','vertex2'])
+        
 
```

Comment:
I made a couple of modifications for input node information based on the description of LocalNeighborhood, not sure if this works fine